schemas/webhooks/webhook_event.yaml

```diff
@@ -130,14 +130,14 @@ webhook_event_object_status:
     # ../reports/definitions.yaml#/report_status
     - awaiting_data
     - awaiting_approval
-    - cancelled
+    # - cancelled (already defined)
```

Comment:
Why not deleting instead of commenting?

Reply:
It was just to not forget same value is defined for other kind of objects also: since status from different object are de facto merged here, it's not easy to know for which object that status make sense.

But if you prefer we can remove such comments.

Reply:
Not a big deal, you can keep it 😁